tasks/db_deploy/db_deploy.py

```diff
@@ -46,17 +46,21 @@ def handler(
     # Get the configuration
     config = get_configuration()
 
-    return task(config)
+    #get partition bucket list
+    partition_bucket = event["buckets"]
```

Comment:
`partition_bucket` -> `partition_buckets`

Reply:
Updated names to make them more clear